inc/editor/blocks/mega-menu.php

```diff
@@ -0,0 +1,85 @@
+<?php
+/**
+ * Block functions to extend the core group block.
+ */
+
+namespace WMF\Editor\Blocks\MegaMenu;
+
+/**
+ * Connect namespace methods to hooks and filters.
+ *
+ * @return void
+ */
+function bootstrap(): void {
+	add_filter( 'render_block', __NAMESPACE__ . '\\render_navigation', 10, 2 );
+}
+
+/**
+ * Make sure main group block on News+ has 'alignwide' class.
+ *
+ * @param string $block_content The block content.
+ * @param array  $block The full block, including name and attributes.
+ * @return $block_content The updated block content.
+ */
+function render_navigation( $block_content, $block ) {
+	if ( 'hm-blocks/hm-mega-menu-block' !== $block['blockName'] && ! in_array( $block['attrs']['className'], [ 'language-switcher', 'search' ], true ) ) {
```

Comment:
I'm getting an error here `Undefined array key "className"` - I can see your comment above about the alignwide class, but perhaps we should guard against it?